xml/System.Diagnostics/Process.xml

```diff
@@ -6805,10 +6805,10 @@ The file specified in the <paramref name="fileName" /> could not be found.
         <Parameter Name="timeout" Type="System.TimeSpan" Index="0" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="timeout">To be added.</param>
-        <summary>Instructs the Process component to wait the specified number of milliseconds for the associated process to exit.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="timeout">The amount of time to wait for the associated process to exit.</param>
+        <summary>Instructs the Process component to wait the specified amount of time for the associated process to exit.</summary>
+        <returns><see langword="true" /> if the associated process has exited; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns>
          <see langword="true" /> if the associated process has exited; otherwise, <see langword="false" />.</returns>
```

Reply:
This is just because of how mdoc would reformat it during the next CI if we left it as-is.